### DISCUSSION TITLE
3/4(월) 야무 쌤 1차 피드백 질문 내용 점검 요청

### DISCUSSION THREAD

**Discussion by DragonTiger92** [FRONTENDSCHOOL8/Book-Kong#114] [Announcements]
야무 쌤께 3/4(월) 10:00~10:45에 있을 1차 피드백 시간에 질문할 내용 정리하였습니다. 확인 후 추가하거나 수정하고 싶은 내용이 있으시다면 아래에 코멘트 달아주시길 바랍니다.

# 3/4(월) 1조 1차 피드백 시간에 야무 쌤께 질문할 내용들

- CORS policy: 알라딘 API로 자료 가져올 때 문제 발생

- 여러 DB의 정보 가져올 때 어떻게 자료를 가져오고 관리하면 좋은지?

- 현재 팀 내에서 설정된 백로그 이슈 우선 순위에 대해 어떻게 생각하시는지 듣고 싶다.

- 우리가 현재 하고 있는 작업 과정에 어떤 문제가 있고, 어떤 걸 보완하면 좋을지에 대해 여쭤보기.

- 책 검색 페이지에서 검색 창에 검색어 입력 시 쿼리에 해당하는 데이터 불러오는 기능 구현할 때, 불러올 데이터가 많으면 어떤 식으로 데이터를 서버에 요청하게 좋을지에 대해서 여쭤보기. 일단 쿼리에 해당하는 데이터를 전부 불러오는지, 아니면 뷰포트에 보이는 것들 정도만 우선 불러오고 사용자가 추가로 자료를 더 보고자 할 경우 그 부분만 추가로 서버에 요청을 하는 방식으로 기능 구현을 하는지. 그리고 둘 중 추천하는 방식으로 구현을 한다면, 그 기능을 어떻게 구현하면 좋을지에 대해서도 조언 요청하기.

- 웹앱 구현 시 따져야 할 접근성, SEO 관련 이슈들

- Zustand 쓰면 국소적으로 상태 정의할 일이 있더라도 useState 안 쓰고 그냥 Zustand의 store hook 사용하나?

- 회원 가입 페이지의 e-mail input component와 password의 input component validation 기능 구현 시, HTML input native 요소에서 제공하는 pattern, maxlength 등의 attributes를 이용하여 validation 기능을 구현하는 건 어떻게 생각하시는지에 대해 여쭤보기.
 그리고 만약 그 생각에 대해 긍정적으로 생각하시고 그 방법이 JSX 구문 내에서도 구현이 가능할 경우, 그 property name들이 HTML attribute name들이랑 똑같은지 확인할 수 있는 reference 아시는 거 있는지 여쭤보기.

조원 분들께 전체적으로 확인 받고 나면 내일 오후 2~3시 즈음으로 해서 야무 쌤께 질문 목록 DM으로 보내려고 합니다. 한번 보시고 내일 오후 2시 전까지 댓글 달아주시면 감사하겠습니다!

**Comment by DragonTiger92** [comment]
내용 추가하였습니다. 피드백 질문 이대로 가도 되는지, 혹시 더 추가할 내용은 없는지 확인 후 오후 1시까지 코멘트 남겨주시면 감사하겠습니다!

**Comment by Samuel-0930** [comment]
저는 추가할 내용 없습니다!

**Comment by juhyuny** [comment]
저도 없습니다 감사합니다!!

**Comment by HHeoGeon** [comment]
저도 없습니다!

**Comment by DragonTiger92** [comment]
3/2(토) 20:30 경 야무 쌤께 1차 피드백 받고 싶은 내용들 DM으로 송신 완료
